src/manifests_workflow/input_manifests.py

```diff
@@ -120,3 +130,24 @@ def write_manifest(self, version, components=[]):
         manifest_path = os.path.join(manifest_dir, f"{self.prefix}-{version}.yml")
         manifest.to_file(manifest_path)
         logging.info(f"Wrote {manifest_path}")
+
+    def add_to_cron(self, version):
+        logging.info(f"Adding new version to cron: {version}")
+        jenkinsfile = self.cron_jenkinsfile()
+        with open(jenkinsfile, "r") as f:
+            data = f.read()
+
+        cron_entry = f"H/10 * * * * %INPUT_MANIFEST={version}/{self.prefix}-{version}.yml;TARGET_JOB_NAME=distribution-build-{self.prefix}\n"
+
+        if cron_entry in data:
+            raise ValueError(f"{jenkinsfile} already contains an entry for {self.prefix} {version}")
+
+        data = data.replace(
+            "parameterizedCron '''\n",
+            f"parameterizedCron '''\n{' ' * 12}{cron_entry}"
```

Comment:
Can you help me understand why do you want to replace the entire parameterized block with `'''\n{' ' * 12}{cron_entry}`.
I understand triple quotes can be used to represent spanning strings, but what is `{' ' * 12}` being used for here?

Thanks.

Reply:
It's just 12 spaces to align things nicely, see https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/check-for-build.jenkinsfile#L7

![Screen Shot 2021-12-27 at 11 35 26 AM](https://user-images.githubusercontent.com/542335/147490623-9c8e5b81-d049-4ca3-8fe5-724bd2592038.png)

Reply:
Thanks.